implementations/rust/ockam/ockam_node/src/context/send_message.rs

```diff
@@ -99,6 +99,36 @@ impl Context {
             .into_body()
     }
 
+    /// Using a temporary new context, send a message
+    ///
+    /// This helper function uses [`new_detached`] and [`send`] internally.
+    /// See their documentation for more details.
+    ///
+    /// [`new_detached`]: Self::new_detached
+    /// [`send`]: Self::send
+    pub async fn send_extended<T>(
```

Comment:
Do you have a particular use case in mind why we would need to create a temporary context to send a message, instead of sending it from the existing mailbox? Like `app` address, or whichever is available from where you're sending the message

Reply:
How would you add the `outgoing_access_control` to the current mailbox?

Reply:
Currently, you can't, but it should be easy to implement this.
I'm actually good with landing this PR, it's a solid change by itself. But if we can avoid creating new mailboxes for the use case you implemented this change, that would be nice.